src/input/pointer/mod.rs

```diff
@@ -742,6 +742,7 @@ impl<D: SeatHandler + 'static> PointerInternal<D> {
                     focus.motion(seat, data, &event);
                 }
                 (focus, Some((old_focus, _))) => {
+                    data.cursor_image(seat, CursorImageStatus::default_named());
```

Comment:
Maybe it would be better to place that inside the default impl of `PointerTarget::replace`? Between the leave and enter of the new target.

Reply:
I guess so. I wanted to keep it local to the other part resetting the image (in the `(focus, None)` case), since I don't really see this as the responsibility of `focus`.

Reply:
imo it makes sense to keep it local to `leave`. by moving it to the default impl downstream could also override this behavior (though I am not sure this will ever be needed).

Reply:
Makes sense. Updated